applications/GeoMechanicsApplication/tests/cpp_tests/test_constitutive_law_set6.cpp

```diff
@@ -0,0 +1,56 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "custom_utilities/constitutive_law_utilities.hpp"
+#include "includes/checks.h"
+#include "testing/testing.h"
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(SetSixConstitutiveParametersCorrectResults, KratosGeoMechanicsFastSuite)
+{
+    ConstitutiveLaw::Parameters ConstitutiveParameters;
+
+    Vector strain_vector(3);
+    strain_vector(0)           = 1.0;
+    strain_vector(1)           = 2.0;
+    strain_vector(2)           = 3.0;
+    Matrix constitutive_matrix = IdentityMatrix(5, 5);
+    Vector N(3);
+    N(0)                                         = 0.1;
+    N(1)                                         = 0.2;
+    N(2)                                         = 0.5;
```

Comment:
And a similar change here:
```suggestion
    N <<= 0.1, 0.2, 0.5;
```

Reply:
fixed